Formula/b/backtrace_crashpad.rb

```diff
@@ -0,0 +1,77 @@
+class BacktraceCrashpad < Formula
+  desc "Backtrace version of Crashpad with file attachment support and other improvements"
+  homepage "https://github.com/backtrace-labs/crashpad"
+  url "https://github.com/backtrace-labs/crashpad.git",
+      tag:      "v0.1.0",
+      revision: "031c47b4b3349d784d330f670360bb8af0266da3"
+  license "Apache-2.0"
+
+  depends_on "cmake" => :build
+
+  # TODO: Creating iOS variants for the client at some point
```

Comment:
That's not a supported platform for Homebrew though

Reply:
Not even for cross-compilation purposes? i.e. we keep the iOS static library variant if somebody wants to build an iOS app for it? 

In any case @shraddhaarora we can probably delete all these TODOs already and just keep internal notes

Reply:
No, and static libraries are generally discouraged in Homebrew since they make no use of security updates by the package manager

Reply:
Makes sense!